src/main/java/browserview/BrowserComponent.java

```diff
@@ -244,7 +246,11 @@ public void showFilterDocs() {
     public void showIssue(String repoId, int id, boolean isPullRequest, boolean isForceRefresh) {
         if (isPullRequest) {
             logger.info("Showing pull request #" + id);
-            runBrowserOperation(() -> driver.get(GitHubURL.getPathForPullRequest(repoId, id), isForceRefresh));
+            runBrowserOperation(() -> driver.get(
+                GitHubURL.getPathForPullRequest(repoId, id),
+                isForceRefresh,
+                (currentUrl, targetUrl) -> currentUrl.equalsIgnoreCase(targetUrl + "/commits") || currentUrl.equalsIgnoreCase(targetUrl + "/files")
+            ));
```

Comment:
Yes it would work. I will shift this check out of `ChomeDriverEx` into here (`BrowserComponent`)